src/components/documentation/new-page.html

```diff
@@ -196,8 +194,29 @@ <h3>Type your documentation here:</h3>
                 </md-input-container>
             </div>
 
+            <div layout-gt-sm="row">
+              <md-input-container ng-if="($ctrl.isMarkdown() || $ctrl.isMarkdownTemplate()) && $ctrl.templates.length > 0" flex-gt-xs>
+                <label>Template</label>
+                <md-select ng-model="$ctrl.selectedTemplate" ng-change="$ctrl.onChangeMarkdownTemplate()">
+                  <md-option  ng-repeat="template in $ctrl.templates"
+                              ng-value="template">
+                    <md-icon md-svg-src="assets/logo_{{template.type | lowercase}}.svg"></md-icon>
+                    <span style="font-style: italic; opacity: 0.5;">{{ template.parentPath }}</span>/{{ template.name }}
+                  </md-option>
+                </md-select>
+                <div class="hint">
+                  Choose a template
+                </div>
+              </md-input-container>
+            </div>
+
+            <div class="gv-page-info-banner" ng-hide="!$ctrl.page.content">
+              <ng-md-icon icon="info" class="gv-info"></ng-md-icon>
+              Selecting a template will override the current documentation.
+            </div>
+
```

Comment:
Could we make it ui-components ?
And you can use gv-icon instead of ng-md-icon

Reply:
issue created here : https://github.com/gravitee-io/gravitee-ui-components/issues/288